pyfar/dsp/filter/band_filter.py

```diff
@@ -450,3 +450,68 @@ def crossover(signal, N, frequency, sampling_rate=None):
         # return the filtered signal
         signal_filt = filt.process(signal)
         return signal_filt
+
+
+def notch(signal, center_frequency, quality, sampling_rate=None):
+    """
+    Create and apply a second order IIR notch filter.
+
+    A notch filter has a magnitude of 0 (-infinity dB) at the center frequency
+    and a magnitude of 1 (0 dB) at almost all other frequencies. This is a
+    wrapper for ``scipy.signal.iirnotch``.
+
+    Parameters
+    ----------
+    signal : Signal, None
+        The Signal to be filtered. Pass ``None`` to create the filter without
+        applying it.
+    center_frequency : number
+        Frequency in Hz at which the magnitude response will be 0
+        (-infinity dB).
+    quality : number
+        The quality is defined at the -3 dB points around the
+        `center_frequency` and is the inverse band width in octaves relative
```

Comment:
What does that have to do with octaves?
I'd again use the scipy description here. 
```suggestion
        The quality factor. Dimensionless parameter that characterizes notch filter -3 dB bandwidth bw relative to its center frequency, `Q = center_frequency/bw`.
```

Reply:
Scipy refers to the bandwidth in Hz (``bw_inHz``), at least for me bandwidth in octaves is more common in the audio community, i.e., ``bw = bw_in_Hz / center_frequency``. That is what I ment to make clear. Changed it to something between.